testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -159,6 +163,25 @@ def test_volume(self):
             err_msg="wrong volume for unitcell (rhombic dodecahedron)")
 
 
+class TestGROIncompleteVels(object):
+    def setUp(self):
+        self.u = mda.Universe(GRO_incomplete_vels)
+
+    def tearDown(self):
+        del self.u
+
+    def test_load(self):
+        assert_(len(self.u.atoms) == 4)
```

Comment:
use `assert_equal(len(self.u.atoms), 4)` --- saves the `assert_` import and is clearer.